src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.tsx

```diff
@@ -43,6 +51,14 @@ const getInitialBundler = (path: string, urls: BundlerOptionUrls) => {
     return BUNDLER_OPTIONS.Rollup
   } else if (path === urls.Webpack) {
     return BUNDLER_OPTIONS.Webpack
+  } else if (path === urls.Remix) {
+    return BUNDLER_OPTIONS.Remix
+  } else if (path === urls.Nuxt) {
+    return BUNDLER_OPTIONS.Nuxt
+  } else if (path === urls.SvelteKit) {
+    return BUNDLER_OPTIONS.SvelteKit
+  } else if (path === urls.SolidStart) {
+    return BUNDLER_OPTIONS.SolidStart
```

Comment:
is it worth it to defensively add a "default" fallback here if for some reason the path isn't one of our options